ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java

```diff
@@ -33,15 +32,15 @@ public void setUp() throws Exception {
     }
 
     @Test
-    @Ignore
     @Category(BuildTests.class)
-    public void EndVisitTest() throws Exception {
+    public void endVisitTest() {
         FindPatientPage findPatientPage = homePage.goToFindPatientRecord();
         findPatientPage.enterPatient(patient.identifier);
+        findPatientPage.waitForPageToLoad();
         ClinicianFacingPatientDashboardPage clinicianFacingPatientDashboardPage = findPatientPage.clickOnFirstPatient();
         PatientVisitsDashboardPage patientVisitsDashboardPage = clinicianFacingPatientDashboardPage.goToRecentVisits();
         patientVisitsDashboardPage = patientVisitsDashboardPage.endVisit();
-        assertNull(patientVisitsDashboardPage.getActiveVisit());
+        assertTrue(patientVisitsDashboardPage.containsText("No active visit"));
```

Comment:
Though the [element](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/6aaadb992e712bdf3e879145bdeece5f282892ae/ui-tests/src/test/java/org/openmrs/reference/page/PatientVisitsDashboardPage.java#L21) that is consumed by  `getActiveVisit()` passes with [startVistTest](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/285dcc90731099bd2d9368097428a21b3c8bcca2/ui-tests/src/test/java/org/openmrs/reference/StartVisitTest.java#L40) when `assertNotNull()` is applied But the element becomes no longer attached to the DOM  when `assertNull()` is applied on `any` of the element`(VISIT_LIST)` lookups [`id(), cssSelector(), xpath(), linkText(), etc` ] Its from that reason that we rather use `assertTrue()` since it achieves the same goal in this context(`case`)